src/components/timers/Tabata.tsx

```diff
@@ -1,3 +1,176 @@
-const Tabata = () => null;
+import { useState, useRef, useEffect } from'react';
+import { PlayPauseButton, FastForwardButton, ResetButton } from '../generic/Button';
+import { Input } from '../generic/Input';
+import { DisplayTime } from '../generic/DisplayTime';
+import { DisplayRounds } from '../generic/DisplayRounds';
+import { convertToMs } from '../../utils/helpers';
+
+const Tabata = () => {
+    const [workMinTimeValue, setWorkMinTimeValue] = useState(0);
```

Comment:
There are a lot of different state hooks here. Consider collapsing some of these together. 

Ex. 
restMinTimeValue and restSecTimeValue could become 
`const [restTimeValue, setRestTimeValue] = useState({minutes: 0, seconds: 0});`